recipes/spgl1/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "spgl1" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spgl1-{{ version }}.tar.gz
+  sha256: a2a524724097bad18dd88a306dbcc99124c6c46ffcbb1a96d6ba6dd6fe2f7404
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - pytest-runner
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.5
+    - numpy >=1.15.0
+    - scipy
+
+test:
+  imports:
+    - spgl1
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/drrelyea/SPGL1_python_port
+  summary: 'SPGL1: A solver for large-scale sparse reconstruction.'
+  license: LGPL-2.1
+  license_file:
+    - License.md
```

```suggestion
    - License.md
    - COPYING
```
Ref: https://github.com/drrelyea/spgl1/blob/master/COPYING